.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: /
+  groups:
+    GitHub_Actions:
+      patterns:
+        - "*"  # open a single pull request to update all actions
```

Comment:
Are the quotes required here?

Reply:
They are, `*` is a special character in YAML. You can use `&` and `*` to declare and reference properties to de-duplicate repeated values.

This:

```yaml
myprop:
  uid: &id XXX
myprop1:
  id: *id
```

Is equivalent to this:

```yaml
myprop:
  uid: XXX
myprop1:
  id: XXX
```